tensorbay/client/version.py

```diff
@@ -271,14 +273,23 @@ def get_draft(self, draft_number: Optional[int] = None) -> Draft:
 
         raise ResourceNotExistError(resource="draft", identification=draft_number)
 
-    def list_drafts(self) -> PagingList[Draft]:
+    def list_drafts(
+        self, status: Optional[str] = "OPEN", branch_name: Optional[str] = None
```

Comment:
since we have use `branch_name` as filter, should we update the usage of `list_drafts` in `GAS.update_dataset`?

Reply:
list_drafts is u